### ISSUE TITLE
Source mapping in minified code

### ISSUE THREAD

**Issue by joshblum** [esvit/ng-table#930]
Hey recently installed this library via npm (version `2.1.0`) and noticed that the minified source is requesting the source mapping. What is the reasoning behind this? I thought that minified code should not contain source maps since it defeats the purpose of minifying. Aside from the inefficiency, the file is also not present from the npm install. (I think related to #505) Is there a reason that the source maps are included with the minified code?

**Comment by christianacca** [comment]
Hi Joshua,

I've just npm installed the latest version (2.1.0) of ng-table, and I DO see the corresponding source maps files (see screenshot below).

![image](https://cloud.githubusercontent.com/assets/886590/19928039/a8ab5952-a0f4-11e6-96c1-1e31161386a4.png)

So the minified files only contain a *link (url) to the source map file*. 

By default, the browser will NOT try and download the source map file using that link. However, when you open say the dev tools in chrome, this will trigger the download of the source map(s).

So in other words, for normal production use, the source maps will not be downloaded.

C

**Comment by joshblum** [comment]
@christianacca thanks for the clarification! So I assume the 404 I get is related to issue #505 not a missing file in the npm package. Are there any plans to fix this bug?

**Comment by christianacca** [comment]
So if you're using the version 2.1.0 from npm then I don't see why you're getting a 404 for the source map file.

I checked one of the official examples: http://ng-table.com: http://s.codepen.io/christianacca/debug/MwxoKm

Source maps are being downloaded correctly for this example. 

You should be able to check this example yourself. Just navigate to the link above in chrome and try debugging the source code - you will see the code showing up in typescript (for example ngTableParams.ts as per screenshot below)

In this example case the ng-table package is being served from unpck (https://unpkg.com/ng-table@2.1.0/bundles/)


![image](https://cloud.githubusercontent.com/assets/886590/19959539/80727b5e-a19f-11e6-8d5e-8341cfdeee24.png)

**Comment by joshblum** [comment]
@christianacca thanks for the help! I think our issue was related to a combination of a weird routing of our static files and webpacking this library into our source.
